tests/deepsparse/transformers/pipelines/helpers.py

```diff
@@ -82,3 +87,53 @@ def _create_tokenizer(model_name):
             tokenizer.pad_token = tokenizer.eos_token
 
         return tokenizer
+
+
+def parse_params(config_path: str) -> Tuple[Optional[Dict], Optional[str]]:
+    # parses the config file provided
+    assert os.path.isfile(config_path), f"config_path {config_path} is not a file"
+    # reads the yaml file
+    with open(config_path, "r") as f:
+        config = yaml.safe_load(f)
+
+    cadence = os.environ.get("CADENCE", "commit")
+    expected_cadence = config["cadence"]
+
+    if not isinstance(expected_cadence, list):
```

Comment:
What do you expect the types to be if its not a list? 
A good way to validate would be to use pydantic where you can do type checks, parse, and serialization

Reply:
This may be a bit of an overkill. Let's not forget that it is a simple dev-facing interface. If not a list, it should be a string. I am implicitly assuming this, maybe to implicitly.